vagrant.json

```diff
@@ -72,7 +76,11 @@
                 ]
             ],
             "boot_command": [
-                "<enter><wait10><wait10><wait10><wait10><wait10><enter><enter>",
+                "<enter><wait10><wait10><wait10><wait10><wait10><enter><enter><wait>",
+                "useradd -m -d /home/vagrant -g root vagrant || /bin/true<enter><wait>",
```

Comment:
I don't think we need this lines. There is no vagrant involved during this step. packer starts your hypervisor (eg. virtualbox), then it will boot up a VM with the ISO attached. After waiting for the root shell on the ISO it will download the install.sh and install-chroot.sh scripts. And the install-chroot.sh script will setup sshd and all necessary users inside of the chroot on the virtualdisk of the VM. Your script setups the vagrant user inside of the ISO environment, this environment is no longer in place when the VM has rebooted.